cdap-api/src/main/java/io/cdap/cdap/api/plugin/PluginConfigurer.java

```diff
@@ -128,15 +128,4 @@
     Map<String, String> properties, MacroEvaluator evaluator, MacroParserOptions options) throws InvalidMacroException {
     throw new UnsupportedOperationException("Evaluating macros is not supported.");
   }
-
-  /**
-   * Creates a new instance of {@link ClassLoader} that contains this program classloader, all the plugins
-   * export-package classloader instantiated by this configurer and system classloader in that loading order.
-   * Currently this method is only supported in services.
-   *
-   * @return a combined classloader
-   */
-  default ClassLoader createClassLoader() {
```

Comment:
this will break dependent projects that use it. Would be better to remove it after the downstream projects are changed.

Reply:
this method is introduced in 6.5 and is only used in connection handler, I have already changed the place that uses it. So it should be safe to just remove it.